boards/imix/debug.gdb

```diff
@@ -0,0 +1,6 @@
+
+target remote :3333
+monitor reset
```

Comment:
Do you know why we `monitor reset` multiple times? I've seen many gdb files that do it, but I've never known why.